tests/test_config.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2021 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+import pytest
+
+from reana_server.config import _is_valid_rate_limit
+
+correct_rate_limit_values = [
+    "20 per second",
+    "100 per minute",
+    "6000 per hour",
+    "20/second",
+]
+
+
+@pytest.mark.parametrize("value", correct_rate_limit_values)
+def test_is_valid_rate_limit_correct(value: str):
+    assert _is_valid_rate_limit(value)
+
+
+incorrect_rate_limit_values = [
+    "20per second",
+    "100 perminute",
+    "20second",
+    "20",
+    "second",
+    "",
+]
+
+
+@pytest.mark.parametrize("value", incorrect_rate_limit_values)
+def test_is_valid_rate_limit_incorrect(value: str):
+    assert not _is_valid_rate_limit(value)
```

Comment:
You can join both tests in one and parametrize the value and the output (e.g. `"value, valid"`).

Reply:
Done